api/scripts/certification/target-profile-to-calibrated-framework.js

```diff
@@ -0,0 +1,46 @@
+import { knex } from '../../db/knex-database-connection.js';
+import { usecases } from '../../src/certification/configuration/domain/usecases/index.js';
+import { ComplementaryCertificationKeys } from '../../src/certification/shared/domain/models/ComplementaryCertificationKeys.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+export class TargetProfileToCalibratedFrameworkScript extends Script {
+  constructor() {
+    super({
+      description: 'Use target profile to create a consolidated framework',
+      permanent: false,
+      options: {
+        complementaryCertificationKey: {
+          type: 'string',
+          describe: 'Certification key',
+          requiresArg: true,
+        },
+        targetProfileId: {
+          type: 'number',
+          describe: 'Target profile ID',
+          requiresArg: true,
+        },
+      },
+    });
+  }
+
+  async handle({ options, logger }) {
+    logger.info(`① Check ${options.complementaryCertificationKey} existence in the domain`);
+    if (!Object.values(ComplementaryCertificationKeys).includes(options.complementaryCertificationKey)) {
+      throw new Error('The certification key is missing');
+    }
+
+    logger.info(`② Retrieve target-profile #${options.targetProfileId} tubes`);
```

Comment:
Pinaillage : les caracteres un peu alternatifs comme ca sur un script en console ca sent les affichages unicode tout petes 😅 #U+D000-DFFF

```suggestion
    logger.info(`Retrieve target-profile #${options.targetProfileId} tubes`);
```